opencensus/exporters/stats/prometheus/internal/prometheus_exporter.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "opencensus/exporters/stats/prometheus/prometheus_exporter.h"
+
+#include <utility>
+#include <vector>
+
+#include "metrics.pb.h"
+#include "opencensus/exporters/stats/prometheus/internal/prometheus_utils.h"
+#include "opencensus/stats/stats.h"
+
+namespace opencensus {
+namespace exporters {
+namespace stats {
+
+std::vector<io::prometheus::client::MetricFamily>
+PrometheusExporter::Collect() {
+  const auto& data = opencensus::stats::StatsExporter::GetViewData();
```

Comment:
Why reference?

Reply:
Changed--no good reason, given that GetViewData returns a value.